docs/reference/filters.md

```diff
@@ -465,6 +466,26 @@ Example:
 * -> repeatContent("I will not waste chalk. ", 1000) -> <shunt>;
 ```
 
+### wrapContent
+
+Add prefix and suffix to the response.
+
+Parameters:
+
+* suffix (string)
+* prefix (string)
```

Comment:
maybe prefix should be before suffix in this list

Reply:
True, fixed